socorro/external/es/super_search_fields.py

```diff
@@ -1448,6 +1449,31 @@ def number_field(
         "query_type": "flag",
         "storage_mapping": {"type": "short"},
     },
+    # NOTE(willkg): This populates the new location in processed_crash with data with
+    # the right type. In 8/2022, we can:
+    #
+    # 1. remove contains_memory_report (deprecated)
+    # 2. rename this one to contains_memory_report
+    # 3. change is_exposed and is_returned to True
+    # 4. change the namespace and in_database_name and remove source/destination keys
```

Comment:
Hopefully future me remembers this and undoes it. I'll probably write up a bug for it.